lib/bike_brigade_web/live/campaign_live/duplicate_campaign_component.ex

```diff
@@ -100,4 +139,37 @@ defmodule BikeBrigadeWeb.CampaignLive.DuplicateCampaignComponent do
       instructions_template: %{body: old_campaign.instructions_template.body }
     })
   end
+
+  defp copy_delivery_tasks(old_campaign, new_campaign) do
+    for old_task <- old_campaign.tasks do
+      task_params = Delivery.Task.fields_for(old_task)
+      {:ok, new_task} = Delivery.create_task_for_campaign(new_campaign, task_params)
+
+      old_task = old_task |> Repo.preload(:task_items)
+
+      for task_item <- old_task.task_items do
+        %Delivery.TaskItem{}
+        |> Delivery.TaskItem.changeset(%{task_id: new_task.id, item_id: task_item.item_id, count: task_item.count})
+        |> Repo.insert()
+      end
```

Comment:
```suggestion
      old_task = old_task |> Repo.preload(:task_items)
      task_params = Delivery.Task.fields_for(old_task)
        |> Map.put(:task_items,  for task_item <- old_task.task_items, do: %{ item_id: task_item.item_id, count: task_item.count})
      {:ok, new_task} = Delivery.create_task_for_campaign(new_campaign, task_params)
      end
```

Reply:
Hmm it wont compile, and I'm getting

```
** (SyntaxError) lib/bike_brigade_web/live/campaign_live/duplicate_campaign_component.ex:147: unexpected comma. Parentheses are required to solve ambiguity in nested calls.

This error happens when you have nested function calls without parentheses. For example:

    one a, two b, c, d

In the example above, we don't know if the parameters "c" and "d" apply to the function "one" or "two". You can solve this by explicitly adding parentheses:

    one a, two(b, c, d)

Or by adding commas (in case a nested call is not intended):

    one, a, two, b, c, d

Elixir cannot compile otherwise. Syntax error before: ','
```

Might just be syntax error, but not sure where to fix?